### ISSUE TITLE
kann nicht builden

### ISSUE THREAD

**Issue by Kuschlsoke** [4ahmns2122-VR/VRProjekt2122-sber#6]
![cant-build](https://user-images.githubusercontent.com/72348412/152485620-eaeb5bdc-195b-43ef-8072-378cdf937eaa.JPG)

**Comment by smeerws** [comment]
Hast du auf build an run gedrückt?

**Comment by Kuschlsoke** [comment]
ja, ich wüsste aber nicht warum es dann nicht builden würde. (run funktioniert natürlich nicht)
gibt es einen anderen weg zu builden?
